en/sql/tuning.rst

```diff
@@ -3829,6 +3829,168 @@ The join with the *right_tbl* table was eliminated.
+
+Queries like Query 1 below, which use inline views, make it easier to understand the content of the query.
+
+.. code-block:: sql
+
+        /* Query 1 */
+        SELECT *
+        FROM (SELECT * FROM athlete WHERE nation_code = 'USA') a,
+        (SELECT * FROM record WHERE medal = 'G') b
+        WHERE a.code = b.athlete_code;
+
+If **View Merging** optimization is not applied, the inline views a and b are processed in advance and stored in temporary storage, 
+and a join operation is performed based on this data.
+
+In this case, the data stored in temporary storage cannot use indexes, resulting in significant performance degradation.
+
+.. code-block:: sql
+
+        /* Query 2 */
+        SELECT *
+        FROM emp a, dept b
+        WHERE a.code = b.athlete_code
+        AND a.nation_code = 'USA'
+        AND b.medal = 'G';
+
+Therefore, the view query block in Query 1 undergoes a merge process with the query block 
+that references the view and is transformed into the form of Query 2.
+
+This is referred to as **View Merging**. Going through this process allows the optimizer to consider more access paths.
+
+In CUBRID, View Merging cannot be performed if the query meets the following conditions: 
+
+#. Using **NO_MERGE** hint on a view.
+
+#. Contains **CONNECT BY**.
+
+#. The view includes a **DISTINCT** clause.
+
+#. **CTE** (Common Table Expressions) is included in the query.
+
+#. Using **OUTER JOIN** with a view.
+
+#. Using aggregation or analytical functions.
+
+#. Using **ROWNUM, LIMIT**, or **GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM()**.
+
+#. Written using **Correlated Subquery**.
+
+#. The view includes methods.
+
+#. The view includes **RANDOM(), DRANDOM(), SYS_GUID()**.
+
+The following is an example that uses **NO_MERGE** hint on a view.
+
+.. code-block:: sql
+
+    SELECT *
+    FROM (SELECT /*+ NO_MERGE*/ * FROM athlete WHERE nation_code = 'USA') a,
+    (SELECT /*+ NO_MERGE*/ * FROM record WHERE medal = 'G') b
+    WHERE a.code = b.athlete_code;
+
+If the **NO_MERGE** hint is used on a view, **View Merging** is not applied.
+
+The following is an example that includes the **CONNECT BY** clause.
+
+.. code-block:: sql
+
+        -- Creating a tree table and then inserting data
+        CREATE TABLE tree(id INT, mgrid INT, name VARCHAR(32), birthyear INT);
+
+        INSERT INTO tree VALUES (1,NULL,'Kim', 1963);
+        INSERT INTO tree VALUES (2,NULL,'Moy', 1958);
+        INSERT INTO tree VALUES (3,1,'Jonas', 1976);
+        INSERT INTO tree VALUES (4,1,'Smith', 1974);
+        INSERT INTO tree VALUES (5,2,'Verma', 1973);
+        INSERT INTO tree VALUES (6,2,'Foster', 1972);
+        INSERT INTO tree VALUES (7,6,'Brown', 1981);
+
+        -- Executing a hierarchical query with CONNECT BY clause
+        SELECT *
+        FROM (SELECT * FROM tree WHERE birthyear = 1973) t
+        CONNECT BY PRIOR t.id=t.mgrid; 
+
+Due to the use of the **CONNECT BY** clause in the above query, **View Merging** cannot be performed.
```

Comment:
It is unclear whether it should be called clauses or statements.
How about not using these words instead?
It is the same in other places as well.
```suggestion
Due to the use of the **CONNECT BY** in the above query, **View Merging** cannot be performed.
```